### PR TITLE
Add/netlify forms

### PR DESCRIPTION
**Pull request by daiki44**
add netlify forms
- [x] news subscribe
https://5ea2557c5734d0000718ec4c--quirky-chandrasekhar-8f7e92.netlify.app/communities/News
- [x] collaboration contact
https://5ea2557c5734d0000718ec4c--quirky-chandrasekhar-8f7e92.netlify.app/communities/Collaboration

### CODE REVIEW

File: src/components/Team/index.js [hunk 1]
```diff
@@ -15,45 +34,23 @@ function Team() {
...
-          <p>
-            Cryptoeconomics Lab's R&D team is looking forward to meet someone
-            who are passionate about blockchain production.
-          </p>
-          {/* TODO: change color */}
-          <a href="https://www.cryptoeconomicslab.com" target="_blank">
-            <Button>Apply</Button>
-          </a>
-        </div>
+        {cards.map((card, i) => (
```
Comment (tkmct) [inline src/components/Team/index.js:37]: you should not use index for key. please use unique property such as name instead.
https://reactjs.org/docs/lists-and-keys.html#keys

File: src/components/News/index.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+          className={styles.iframe}
+          src="https://www.youtube.com/embed/St1-YmucrTM"
+          frameBorder="0"
+          allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+          allowFullScreen
+        ></iframe>
+        <p>
+          <a
+            href="https://www.youtube.com/channel/UCp1w3jQNBDD2Pz4kool36Pw"
+            target="_blank"
```
Comment (tkmct) [inline src/components/News/index.js:38]: you should use `rel="noopener"` along with `target="_blank"` for security.
FYI https://developers.google.com/web/tools/lighthouse/audits/noopener?hl=ja

File: src/components/Form/ContactForm/index.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+function ContactForm(props) {
+  return (
+    <form
+      className={styles.root}
+      action="/contact/thanks"
+      name="contact"
+      method="POST"
+      data-netlify="true"
+      {...props}
```
Comment (tkmct) [inline src/components/Form/ContactForm/index.js:13]: It's not a good practice to pas spreaded props
https://reactjs.org/docs/jsx-in-depth.html#spread-attributes

**Comment by daiki44** [comment]
fixed!

**Pull request merged**